rclcpp_generic/include/rclcpp_generic/generic_publisher.hpp

```diff
@@ -0,0 +1,88 @@
+// Copyright 2018, Bosch Software Innovations GmbH.
```

Comment:
The license was copied from `rosbag2_transport`. Should I just leave it like this?

Reply:
That should be fine